src/extra.i

```diff
@@ -3320,35 +3370,28 @@ PyObject* extractWORDS(mupdf::FzStextPage& this_tpage, PyObject *delimiters)
                                 );
                     }
                     mupdf::fz_clear_buffer(buff);
-                    buflen = 0;  // reset char counter
-                    if (word_delimiter) continue;
+                    buflen = 0;
                 }
```

Comment:
Could we use the size of `buff`'s data here (e.g. from `fz_buffer_storage()`) instead of maintaining this separate measure? I guess there could be a small overhead, but it would be simpler, clearer and less error-prone.